go.mod

```diff
@@ -4,50 +4,28 @@ go 1.13
-	github.com/emicklei/go-restful-swagger12 v0.0.0-20170208215640-dcef7f557305 // indirect
-	github.com/evanphx/json-patch v4.2.0+incompatible // indirect
-	github.com/gogo/protobuf v1.1.1 // indirect
 	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
-	github.com/google/btree v1.0.0 // indirect
-	github.com/gorilla/websocket v1.4.0 // indirect
-	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 // indirect
-	github.com/grpc-ecosystem/grpc-gateway v1.9.2 // indirect
-	github.com/imdario/mergo v0.3.6 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/jonboulle/clockwork v0.1.0 // indirect
-	github.com/json-iterator/go v1.1.5 // indirect
-	github.com/kubernetes-incubator/custom-metrics-apiserver v0.0.0-20190918110929-3d9be26a50eb
-	github.com/kubernetes-incubator/metrics-server v0.0.0-20190416094704-020262240108
-	github.com/munnerz/goautoneg v0.0.0-20190414153302-2ae31c8b6b30 // indirect
-	github.com/onsi/ginkgo v1.6.0
-	github.com/onsi/gomega v1.4.2
-	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/common v0.0.0-20181126121408-4724e9255275
-	github.com/sirupsen/logrus v1.4.2 // indirect
-	github.com/soheilhy/cmux v0.1.4 // indirect
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.2 // indirect
-	github.com/stretchr/testify v1.3.0
+	github.com/kubernetes-incubator/custom-metrics-apiserver v0.0.0-20200323093244-5046ce1afe6b
+	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/gomega v1.7.0
+	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/common v0.4.1
+	github.com/spf13/cobra v0.0.5
+	github.com/stretchr/testify v1.4.0
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
-	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
-	go.etcd.io/bbolt v1.3.3 // indirect
-	golang.org/x/time v0.0.0-20180412165947-fbb02b2291d2 // indirect
-	google.golang.org/genproto v0.0.0-20180924164928-221a8d4f7494 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	gopkg.in/natefinch/lumberjack.v2 v2.0.0-20170531160350-a96e63847dc3 // indirect
-	gopkg.in/yaml.v2 v2.2.4
-	k8s.io/api v0.0.0-20191004120003-3a12735a829a
-	k8s.io/apimachinery v0.0.0-20191004115701-31ade1b30762
-	k8s.io/apiserver v0.0.0-20191009121109-e383d20f5094 // indirect
-	k8s.io/client-go v0.0.0-20191004120415-b2f42092e376
-	k8s.io/component-base v0.0.0-20191004121406-d5138742ad72
-	k8s.io/klog v0.3.1
-	k8s.io/kube-openapi v0.0.0-20190418160015-6b3d3b2d5666 // indirect
-	k8s.io/metrics v0.0.0-20191004123503-ae3d6ea895be
-	k8s.io/utils v0.0.0-20190308190857-21c4ce38f2a7 // indirect
+	gopkg.in/yaml.v2 v2.2.8
+	k8s.io/api v0.17.3
+	k8s.io/apimachinery v0.17.3
+	k8s.io/client-go v0.17.3
+	k8s.io/component-base v0.17.3
+	k8s.io/klog v1.0.0
+	k8s.io/metrics v0.17.3
+	sigs.k8s.io/metrics-server v0.3.7
+)
+
+replace (
+	// forced by the inclusion of sigs.k8s.io/metrics-server's use of this in their go.mod
```

Comment:
Is this something we can open an issue for in the metrics-server, this seems like an awkward workaround.